### PR TITLE
Added Hub 2 Temperature and Humidity Support. 

### PR DESCRIPTION
**Pull request by alvst**

### CODE REVIEW

File: config.schema.json [hunk 1]
```diff
@@ -203,6 +208,25 @@
                     "functionBody": "return (model.options && model.options.devices && model.options.devices[arrayIndices].deviceId && !model.options.devices[arrayIndices].hide_device && (model.options.devices[arrayIndices].connectionType === 'BLE' || model.options.devices[arrayIndices].connectionType === 'BLE/OpenAPI'));"
                   }
                 },
+                "hub": {
+                  "type": "object",
+                  "properties": {
+                    "hide_temperature": {
+                      "title": "Hide Meter's Temperature Sensor",
```
Comment (donavanbecker) [inline config.schema.json]: Can you rename this to Hide Hub?

File: config.schema.json [hunk 2]
```diff
@@ -203,6 +208,25 @@
...
+                      "type": "boolean",
+                      "condition": {
+                        "functionBody": "return (model.options && model.options.devices && !model.options.devices[arrayIndices].hide_device && (model.options.devices[arrayIndices].configDeviceType === 'Hub 2') && model.options.devices[arrayIndices].deviceId);"
+                      }
+                    },
+                    "hide_humidity": {
+                      "title": "Hide Meter's Humidity Sensor",
```
Comment (donavanbecker) [inline config.schema.json]: Same here

**Review by donavanbecker — CHANGES_REQUESTED** [review]
Fixes those two and then I will merge.

**Review by alvst — COMMENTED** [review]
Both changes you requested have been resolved. Need to improve my process, my auto-formatter is very different from yours so I developed all my changes then had to figure out the best way to get them back into here which I did by copy and pasting and then manually matching the style.

**Pull request merged**